index.html

```diff
@@ -18,14 +18,14 @@
   <div class="wrapper">
     <header>
       <nav>
-        <a href="#">
+        <a href="#" tabindex="0">
```

Comment:
- Great job for updating most accessibility, I think you missed updating the color contrast [here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-28-at-18-28-36-863?failing) is your result, please update based on this [checker](https://www.aditus.io/button-contrast-checker/) :+1: 
![Screenshot from 2023-02-28 21-33-56](https://user-images.githubusercontent.com/52010789/221947234-b2dfdd34-2645-42ae-aade-6a07ffc89198.png)
- [Optional] Kindly your logo is not part of your navigtion so make it outside `nav ` tag for semantic and user accesiblity reasons:+1: